src/inc/msquic.h

```diff
@@ -890,6 +890,8 @@ typedef struct QUIC_SCHANNEL_CREDENTIAL_ATTRIBUTE_W {
 #define QUIC_PARAM_CONN_STATISTICS_V2                   0x05000016  // QUIC_STATISTICS_V2
 #define QUIC_PARAM_CONN_STATISTICS_V2_PLAT              0x05000017  // QUIC_STATISTICS_V2
 #define QUIC_PARAM_CONN_ORIG_DEST_CID                   0x05000018  // uint8_t[]
+#define QUIC_PARAM_CONN_THIS_RELIABLE_RESET_ENABLED     0x05000019  // uint8_t (BOOLEAN)
```

Comment:
I think "THIS" is "LOCAL" if you really need these two settings

Reply:
got rid of 'this' lol